<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>requestLanguageDialog</class>
 <widget class="QDialog" name="requestLanguageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>77</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxLanguage">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonYes">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
